src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $cardWigth;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  line-height: 18px;
+
+  &__monoblock_picture {
+    width: 160px;
+    height: 134px;
+  }
+  
+  &__name_item {
+    font-weight: 500;
+    margin-top: 40px;
+    color: #060b35;
+  }
+  
+  &__code_item {
+    display: flex;
+    justify-content: space-between;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 4px;
+    color: $mainColor;
+    width: 100%;
+  }
+  
+  &__rate {
+    margin-top: 16px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    line-height: 14px;
+  }
+  
+  &__text {
+    display: block;
+    margin-top: 2px;
+    font-size: 10px;
+    height: 10px;
+  }
+  
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    width: 100%;
+  }
+  
+  &__price_text {
+    font-size: 12px;
+    color: $mainColor;
+  }
+  
+  &__price_numbers {
+    font-weight: bold;
+    font-size: 16px;
+    color: #060b35;
+  }
+  
+  &__button_buy {
+    margin-top: 16px;
+    width: 100%;
+  }
+  
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+    line-height: 16px;
+    height: 40px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    box-sizing: border-box;
+    transition: 0.3s ease-in-out;
+    text-transform: uppercase;
+  }
+  
+  &__link:hover {
+    color: #00acdc;
+    background-color: #fff;
+    border-radius: 5px;
```

Comment:
border-radius don't change on hover state, delete it

Reply:
ok

Reply:
You didn't delete border-radius for hover state